src/qibolab/dummy.py

```diff
@@ -0,0 +1,122 @@
+    "nqubits": 2,
+    "description": "Dummy 2-qubits platform.",
+    "qubits": [
+        0,
+        1,
+    ],
+    "settings": {"sampling_rate": 1000000000, "relaxation_time": 0, "nshots": 1024},
+    "resonator_type": "2D",
+    "topology": [[0, 1]],
+    "native_gates": {
+        "single_qubit": {
+            0: {
+                "RX": {
+                    "duration": 40,
+                    "amplitude": 0.005,
+                    "frequency": 4700000000,
+                    "shape": "Gaussian(5)",
+                    "type": "qd",
+                    "start": 0,
+                    "phase": 0,
+                },
+                "MZ": {
+                    "duration": 1000,
+                    "amplitude": 0.0025,
+                    "frequency": 7226500000,
+                    "shape": "Rectangular()",
+                    "type": "ro",
+                    "start": 0,
+                    "phase": 0,
+                },
+            },
+            1: {
+                "RX": {
+                    "duration": 40,
+                    "amplitude": 0.0484,
+                    "frequency": 4855663000,
+                    "shape": "Drag(5, -0.02)",
+                    "type": "qd",
+                    "start": 0,
+                    "phase": 0,
+                },
+                "MZ": {
+                    "duration": 620,
+                    "amplitude": 0.003575,
+                    "frequency": 7453265000,
+                    "shape": "Rectangular()",
+                    "type": "ro",
+                    "start": 0,
+                    "phase": 0,
+                },
+            },
+        },
+        "two_qubit": {
+            "0-1": {
+                "CZ": [
+                    {
+                        "duration": 30,
+                        "amplitude": 0.055,
+                        "shape": "Rectangular()",
+                        "qubit": 0,
+                        "relative_start": 0,
+                        "type": "qf",
+                    },
+                    {"type": "virtual_z", "phase": -1.5707963267948966, "qubit": 1},
+                    {"type": "virtual_z", "phase": -1.5707963267948966, "qubit": 0},
+                ]
+            },
+        },
+    },
+    "characterization": {
+        "single_qubit": {
+            0: {
+                "readout_frequency": 7226500000,
+                "drive_frequency": 0.0,
+                "T1": 0.0,
+                "T2": 0.0,
+                "sweetspot": 0.0,
+                "threshold": 0.0,
+                "iq_angle": 0.0,
+            },
+            1: {
+                "readout_frequency": 7453265000,
+                "drive_frequency": 4855663000,
+                "T1": 0.0,
+                "T2": 0.0,
+                "sweetspot": -0.047,
+                "threshold": 0.00028502261712637096,
+                "iq_angle": 1.283105298787488,
+            },
+        }
+    },
+}
+
+
+def create_dummy(runcard=RUNCARD):
```

Comment:
I would remove this argument because in principle this is a static toy platform.
Moreover, in the future, the `Platform` object should not receive the `runcard` as argument.

Reply:
Thanks @scarrazza. I fixed the issue.